src/LiveComponent/assets/src/morphdom.ts

```diff
@@ -40,7 +40,10 @@ export function executeMorphdom(
                 return true;
             }
 
-            if (!(fromEl instanceof HTMLElement) || !(toEl instanceof HTMLElement)) {
+            if (
+                !(fromEl instanceof HTMLElement || fromEl instanceof SVGElement) ||
+                !(toEl instanceof HTMLElement || toEl instanceof SVGElement)
+            ) {
```

Comment:
Nice find. SVG's not being part of `HTMLElement` is such a pain in the butt :). Thanks for the test case to prevent it from slipping through in the future.